sharepoint/sharepoint-ps/sharepoint-online/Get-SPOEnterpriseAppInsightsReport.md

```diff
@@ -0,0 +1,118 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: 
+applicable: SharePoint Online
```

Comment:
this needs a value. see other files for examples

Reply:
made the required change, marking it as resolved.